background/background.go

```diff
@@ -51,7 +52,7 @@ type Job struct {
 	RetriedAt  time.Time
 }
 
-// Run will contiuously poll a daemon and run the job on the worker.
+// Run will continuously poll a daemon and run the job on the worker.
```

Comment:
Actually it's pronounced continu

![](http://www.livbit.com/article/wp-content/uploads/2008/12/walrus2.jpg)